.github/workflows/build_pr_cache.yml

```diff
@@ -0,0 +1,66 @@
+# Simple workflow for building sccache for PR testing
+name: Build cache for PR testing
+# Note this will currently create a new sscache file and this must be manually pruned until such time as we have a job
+# for this.
+on:
+  # Uncomment to test this path on a pull request
+  # pull_request:
+  #   paths:
+  #     .github/workflows/build_pr_cache.yml
+  #     .github/do_build_ock/**  
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/build_pr_cache.yml
+      - .github/actions/do_build_ock/**
+      - .github/actions/setup_ubuntu_build/**
+  
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build_ca_host_x86_64:
```

Comment:
This job is called 'host' but also runs riscv?

Reply:
bad naming, I'll update